content/docs/2.17/scalers/rabbitmq-queue.md

```diff
@@ -74,6 +74,7 @@ TriggerAuthentication CRD is used to connect and authenticate to RabbitMQ:
 
 > See the [RabbitMQ Ports](https://www.rabbitmq.com/networking.html#ports) section for more details on how to configure the ports.
 
+- `vhostName` - Vhost to use for the connection, overrides any vhost set in the connection string from `host`/`hostFromEnv`. (Optional / Required if Azure AD Workload Identity authorization is used)
```

Comment:
I think that we have to include a new line here so split both sections

<img width="995" alt="image" src="https://github.com/user-attachments/assets/e641b8bd-4d3f-418d-b8f2-fcbfc4eef05f" />

```suggestion
- `vhostName` - Vhost to use for the connection, overrides any vhost set in the connection string from `host`/`hostFromEnv`. (Optional / Required if Azure AD Workload Identity authorization is used)


```

Reply:
Thanks Jorge for the review